packages/expo-secure-store/ios/SecureStoreExceptions.swift

```diff
@@ -6,6 +6,12 @@ internal class InvalidKeyException: Exception {
   }
 }
 
+internal class MissingPlistKeyException: Exception {
+  override var reason: String {
+    "You must set `NSFaceIDUsageDescription` in your info.plist. This is not available in Expo go"
```

Comment:
Small suggestion to make the message more informative and remove that it's Expo Go – it may happen in other apps too (if the config plugin is not added?).

```suggestion
    "You must set `NSFaceIDUsageDescription` in your Info.plist file to use the `requireAuthentication` option"
```